spec/features/search/access_search_page_spec.rb

```diff
@@ -15,12 +15,35 @@
     expect(page).to have_field("search")
   end
 
-  it "renders a list when @query is not empty" do
+  it "renders a list of persons when @query is not empty" do
     FactoryBot.create :person
     FactoryBot.create(:person, first_name: 'Micha', last_name: 'Perscheid', title: 'Not doctor')
     visit "#{search_path}?query=Mich&commit=Search"
     expect(page).to have_link 'Dr. Michael Perscheid'
     expect(page).to have_link 'Not doctor Micha Perscheid'
   end
 
+  it "renders list with rooms containing searched room name" do
+    FactoryBot.create :room
+    FactoryBot.create(:room, full_name: 'H-E.41')
+    visit "#{search_path}?query=H-E&commit=Search"
+    expect(page).to have_link 'H-E.41'
+    expect(page).to have_link 'H-E.42'
+  end
+
+  it "renders list with chairs containing searched chair name" do
+    FactoryBot.create :chair
+    FactoryBot.create(:chair, name: 'Raumschiff Enterprise')
+    visit "#{search_path}?query=Enterprise&commit=Search"
+    expect(page).to have_link 'Enterprise Platform and Integration Concepts'
```

Comment:
```suggestion
```

Reply:
For me it is not obvious how the Enterprise Platforms Chair has anything to do with the test: The test would fail if the EPIC chair closes and is removed from the database. But that isn't what we need to test here right?

Reply:
In this test we should only expect "Raumschiff Enterprise" to be there

Reply:
same would go for "H-E.42" in the test above

Reply:
When we search for 'Enterprise', we should get all results that include this string. This is why we expect the page to show the 'Enterprise Platform and Integration Conceps' chair and 'Raumschiff Enterprise'